test/e2e/basic_test.go

```diff
@@ -10,23 +10,39 @@ import (
 )
 
 var _ = Describe("Test IAM policy", func() {
-	complianceTests := func(policyName, additionalBindingsYaml string) {
+	complianceTests := func(policyName, policyParentName, additionalBindingsYaml, expectedMsg string) {
 		It("Should become compliant at first", func() {
+			By("Checking policy status")
 			Eventually(func() interface{} {
 				pol := GetWithTimeout(gvrIamPolicy, policyName, testNamespace, true)
 				comp, _, _ := unstructured.NestedString(pol.Object, "status", "compliant")
 
 				return comp
 			}, defaultTimeoutSeconds, 1).Should(Equal("Compliant"))
+
+			By("Checking parent policy events")
+			compParentEvents := GetMatchingEvents(testNamespace, policyParentName,
+				"policy: "+testNamespace+"/"+policyName, "^Compliant;")
+			Expect(compParentEvents).NotTo(BeEmpty())
+			nonCompParentEvents := GetMatchingEvents(testNamespace, policyParentName,
+				"policy: "+testNamespace+"/"+policyName, "^NonCompliant;")
+			Expect(nonCompParentEvents).To(BeEmpty())
```

Comment:
Should this be wrapped in `Eventually()`/`Consistently()`?

Reply:
Good idea!